Restore "What packet information to track and send (#1939)" (#1956)
quicwg/base-drafts: draft-ietf-quic-recovery.md

may cause spurious retransmits, but the sender will continue making forward
progress.

## Tracking Sent Packets {#tracking-sent-packets}

QUIC implementations store information about every packet sent. It is expected
implementations will index this information by packet number and crypto context
and store the per-packet fields detailed below for loss recovery and congestion
control.

If a packet is lost, the QUIC transport needs to recover from that loss, such
as by retransmitting the data, sending an updated frame, or abandoning the
frame.  For more information, see Section 13.2 of {{QUIC-TRANSPORT}}.

> Packets MUST be tracked until acknowledged or lost in order to correctly

Reviewer: ```suggestion
Every sent packet MUST be tracked until it is acknowledged or lost to correctly
```

may cause spurious retransmits, but the sender will continue making forward
progress.

## Tracking Sent Packets {#tracking-sent-packets}

> QUIC stores information about every packet sent. It's expected implementations

Reviewer: ```suggestion
QUIC implementations store information about every packet sent. It is
expected that implementations
```

> will index this information by packet number and crypto context and store the

Reviewer: ```suggestion
will index this information by packet number and crypto context and store
```

> per-packet fields detailed below for loss recovery and congestion control.

Reviewer: ```suggestion
per-packet fields ({{sent-packet-fields}}) for loss recovery and congestion control.
```

may cause spurious retransmits, but the sender will continue making forward
progress.

## Tracking Sent Packets {#tracking-sent-packets}

QUIC implementations store information about every packet sent. It is expected
implementations will index this information by packet number and crypto context
and store the per-packet fields detailed below for loss recovery and congestion
control.

> Retrasnmittable packets MUST be tracked until acknowledged or lost in order to

Reviewer: Move this first sentence up and replace the first sentence of the section.
